android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -1521,6 +1522,13 @@ public void onMetadata(@NonNull Metadata metadata) {
         eventEmitter.timedMetadata(metadataArray);
     }
 
+    public void onCues(CueGroup cueGroup) {
+        if (!cueGroup.cues.isEmpty() && cueGroup.cues.get(0).text != null) {
```

Comment:
Don't you think adding following safety check would be a good idea ? 
```suggestion
        if (cueGroup != null && !cueGroup.cues != null && !cueGroup.cues.isEmpty() && cueGroup.cues.get(0).text != null) {
```

Reply:
I totally agree with adding these safety checks; they’re a good idea. I think there might be a small typo in the suggestion, though. Shouldn’t `!cueGroup.cues != null` be `cueGroup.cues != null`?

```java
if (cueGroup != null && cueGroup.cues != null && !cueGroup.cues.isEmpty() && cueGroup.cues.get(0).text != null) {
```

Reply:
> I totally agree with adding these safety checks; they’re a good idea. I think there might be a small typo in the suggestion, though. Shouldn’t `!cueGroup.cues != null` be `cueGroup.cues != null`?
> 
> ```java
> if (cueGroup != null && cueGroup.cues != null && !cueGroup.cues.isEmpty() && cueGroup.cues.get(0).text != null) {
> ```
> 

And I am not sure it passes the linter 😅

Reply:
When I tried doing these checks, the suggestion says to simplify to what I had originally? 🙈
![image](https://github.com/react-native-video/react-native-video/assets/75731066/8a969b1c-9590-4b09-ba59-834827fd8368)

> And I am not sure it passes the linter 😅

which linter? I ran the `check-android` script and it had changes for an unrelated file; but, not this one? 

How would you like me to proceed? 🫡

Reply:
I didn't push these since these automated changes are unrelated to this PR.

![image](https://github.com/react-native-video/react-native-video/assets/75731066/a85c56c3-e983-4927-9415-40bdd026f2e2)

Reply:
OK good, let's merge like this so !

Reply:
sweet, that sounds awesome 🤩